ovh/data_source_ovh_me_ssh_key_test.go

```diff
@@ -0,0 +1,37 @@
+package ovh
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccMeSshKeyDataSource_basic(t *testing.T) {
+	sshKey := acctest.RandomWithPrefix(test_prefix)
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckCredentials(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccMeSshKeyDatasourceConfig, sshKey),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"data.ovh_me_ssh_key.key_1", "key_name", sshKey),
+				),
```

Comment:
i think that it would be better if we'd check the value of the key instead of the keyname

Reply:
Done